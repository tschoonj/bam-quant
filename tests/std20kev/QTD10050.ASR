File: C:\xrfdaten\2014KW30\chris\std20kev\std10050.spe
Start: 21.07.2014 18:26:25
Stop: 21.07.2014 18:28:05
BESSY: 298.7295718 298.4274168
IONI: 2.954792E-11 2.94154E-11
ScanMotor1 exp_x: 1 1
ScanMotor2 exp_y: 1 1
Live: 97811
Run: 100000
Dead: 2
ICR: 1261
OCR: 0
Accumulated BESSY: 59686.3856816
Accumulated Ioni: 5.89122097E-09
N Accumulation: 200
Normfactor:  3.4708591e+008
E_cal:      0.020000000
E_cal_offset:      0.000000
Comments: Sn
exp_x 151.260406494141 151.2604
exp_y 92.3237991333008 92.3238
exp_z 53.8899993896484 53.89
exp_w 135 135.0001
exp_Tisch 870.6 870.600001928
dcm_theta 0 0
det_y 52.85 52.85
dmm_energy 20 20
dcm_energy 23 -184976.74536672
S3_horSize 0.03 3.00000000000001E-02
S3_verSize 0.03 3.00000000000001E-02
$PEAKS:   
      1
  50   2      3.443     42670.       195.       7.74
