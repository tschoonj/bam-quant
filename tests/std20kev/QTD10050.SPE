$DATA:
0 2000
22993
14722
6038
1833
322
44
8
0
0
0
0
0
0
0
0
0
6
39
43
47
37
42
43
56
25
36
45
27
37
33
22
33
38
25
38
35
40
24
35
26
32
26
31
26
25
31
38
30
30
26
30
27
22
17
36
19
28
21
25
19
22
25
23
37
30
43
26
29
21
29
30
24
30
25
22
31
32
27
24
37
43
47
75
99
115
99
59
52
44
33
38
35
44
46
74
92
78
85
66
54
64
56
38
37
40
38
45
47
40
34
38
35
31
30
37
29
38
24
42
38
37
35
44
27
41
31
36
39
46
51
50
55
51
43
38
44
40
44
45
40
52
61
51
62
107
150
164
240
186
233
219
267
216
178
146
116
100
103
103
116
158
165
212
281
417
796
1493
2550
4006
5451
6249
6260
5332
3927
2588
1639
1194
1308
2110
3007
4109
5092
5586
5643
5249
4529
3703
2907
2117
1524
1244
1140
1141
1194
1177
954
784
499
262
201
125
137
171
234
224
255
232
179
102
75
75
55
114
122
171
216
253
234
205
148
126
79
45
39
18
14
12
7
4
8
10
9
8
7
9
8
8
5
11
6
7
10
7
15
7
13
4
7
5
6
9
12
14
11
10
10
6
6
7
5
10
7
9
10
16
15
10
14
6
7
13
7
8
6
9
11
8
9
10
14
9
11
8
14
12
15
16
14
7
9
16
11
5
16
14
8
14
16
8
10
5
9
7
9
4
10
5
8
13
10
5
8
15
12
24
19
17
22
15
15
16
14
5
13
12
12
11
12
10
20
8
20
13
9
15
15
12
14
11
12
10
12
9
11
21
8
7
19
11
11
13
13
12
10
6
23
11
14
7
15
11
10
18
13
13
18
10
19
13
12
26
13
16
17
21
11
11
15
19
16
14
17
12
18
12
14
21
13
9
9
16
13
13
17
19
25
21
22
27
32
13
12
17
19
17
13
16
10
19
12
12
11
14
18
13
13
19
15
23
16
12
18
15
17
17
8
18
13
20
15
24
17
13
17
11
15
15
15
18
17
20
26
15
10
14
7
22
15
17
11
19
11
16
16
15
12
15
7
11
18
10
15
12
12
16
12
15
12
12
11
18
15
9
13
16
11
15
15
19
22
13
10
7
12
15
18
18
17
13
17
17
13
6
11
15
10
11
16
21
19
19
12
7
9
14
16
19
14
20
8
21
13
21
19
12
14
19
17
21
18
16
23
14
16
21
16
7
19
10
18
21
10
15
11
18
18
16
10
18
11
24
11
17
12
15
15
16
13
12
15
11
18
10
17
10
18
15
16
13
14
16
11
15
21
13
16
28
14
7
14
18
14
11
12
15
11
6
15
18
10
28
6
14
21
12
17
11
17
13
16
19
12
9
13
16
17
14
12
10
13
18
6
14
18
16
16
18
13
19
17
16
8
7
11
16
15
16
10
20
15
13
18
16
8
17
17
16
16
14
17
17
19
15
14
11
12
20
6
25
12
20
12
16
15
14
16
19
13
15
12
16
16
14
16
11
15
22
13
15
12
12
19
11
15
14
9
10
16
9
13
17
14
13
9
13
13
12
25
9
15
11
14
14
11
8
15
12
20
13
18
19
18
10
16
3
15
10
13
18
13
15
15
12
10
8
11
21
11
16
14
6
11
11
17
13
11
12
16
12
17
17
10
15
11
13
16
13
16
16
17
17
10
13
14
9
16
19
12
11
21
12
11
20
12
11
10
10
13
13
12
12
12
14
11
12
10
13
12
14
5
16
12
15
19
14
13
14
31
16
6
27
24
34
26
25
34
29
46
49
42
65
43
55
62
39
33
24
30
24
25
11
20
12
9
10
5
5
4
5
3
4
7
5
6
7
3
7
7
7
3
2
1
5
4
5
3
5
5
2
8
3
3
2
6
4
1
3
4
4
4
2
2
1
5
2
3
3
5
2
2
6
6
5
1
2
4
4
3
3
5
4
6
3
7
6
8
8
5
6
5
3
7
6
5
3
7
5
7
7
13
8
11
10
14
6
12
16
18
16
12
9
7
7
6
10
11
5
12
9
9
8
9
5
12
14
10
6
7
10
9
7
15
7
13
10
5
11
12
8
5
11
11
13
20
20
12
14
14
15
20
13
13
24
11
14
14
16
19
18
18
17
23
16
18
23
23
19
30
32
22
22
28
26
18
22
25
23
26
35
33
50
25
21
32
40
41
38
38
44
63
43
33
53
45
39
51
62
44
44
54
59
70
76
82
69
76
106
98
137
166
160
188
210
244
275
300
340
314
387
332
359
348
339
343
316
282
262
302
251
227
216
174
141
130
96
87
61
49
42
29
20
10
13
7
7
4
2
2
2
0
0
0
2
0
2
0
1
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
1
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
2
0
2
0
4
3
2
8
5
2
6
6
7
5
7
4
9
12
13
11
6
10
3
5
7
6
3
6
4
7
2
4
0
0
3
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
1
1
0
0
1
2
1
2
2
2
3
1
3
0
2
1
2
0
1
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
1
0
0
0
0
1
0
1
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
File: C:\xrfdaten\2014KW30\chris\std20kev\std10050.spe
Start: 21.07.2014 18:26:25
Stop: 21.07.2014 18:28:05
BESSY: 298.7295718 298.4274168
IONI: 2.954792E-11 2.94154E-11
ScanMotor1 exp_x: 1 1
ScanMotor2 exp_y: 1 1
Live: 97811
Run: 100000
Dead: 2
ICR: 1261
OCR: 0
Accumulated BESSY: 59686.3856816
Accumulated Ioni: 5.89122097E-09
N Accumulation: 200
Normfactor:  3.4708591e+008
E_cal:      0.020000000
E_cal_offset:      0.000000
Comments: Sn
 
exp_x 151.260406494141 151.2604
exp_y 92.3237991333008 92.3238
exp_z 53.8899993896484 53.89
exp_w 135 135.0001
exp_Tisch 870.6 870.600001928
dcm_theta 0 0
det_y 52.85 52.85
dmm_energy 20 20
dcm_energy 23 -184976.74536672
S3_horSize 0.03 3.00000000000001E-02
S3_verSize 0.03 3.00000000000001E-02
$PEAKS:   
      1
  50   2      3.443     42670.       195.       7.74
