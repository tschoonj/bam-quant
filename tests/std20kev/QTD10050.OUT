     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ  AXIL IBM-PC V3.00                       10-08-2014   14:23:50  บ   
     บ  Spectrum: QTD10050.SPE                                     0.s บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ  Fitting Region: channels 100 -1980        4 iterations done    บ   
     บ  ChiSquare =    11.9   last change =   .02%    lambda= 1.E-06   บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ                        CALIBRATION  DATA                        บ   
     ฬออออออออออออออออหออออออออออออออออออออออออหอออออออออออออออออออออออน   
     บ                บ   Initial estimate     บ    Final estimate     บ   
     ฬออออออออออออออออฮออออออออออออออออออออออออฮอออออออออออออออออออออออน   
     บ  ZERO (eV)     บ      -46.8 ๑   100.0   บ     -46.8 ๑     2.9   บ   
     บ  GAIN (eV/ch)  บ     20.344 ๑   2.000   บ    20.344 ๑    .016   บ   
     บ det NOISE (eV) บ       69.5 ๑    40.0   บ      69.5 ๑    12.6   บ   
     บ  FANO factor   บ       .151 ๑    .050   บ      .151 ๑    .016   บ   
     ศออออออออออออออออสออออออออออออออออออออออออสอออออออออออออออออออออออผ   
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ                           PEAK  DATA                            บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ  # Line       E(KeV)  rel. int.   peak area   st. dev    chi-sq บ   
     บ               chan#   fwhm (eV)      backgr            tot. abs บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ                                                                 บ   
     บ  1 Sn-L1                             17353. ๑    179.           บ   
     บ    L1M3        3.750    .49267        8549. ๑     89.    226.8  บ   
     บ              186.629    130.07         139.           6.58E-01  บ   
     บ    L1M2        3.708    .29965        5200. ๑     55.    211.1  บ   
     บ              184.565    129.54         140.           6.49E-01  บ   
     บ    L1N3        4.377    .19111        3316. ๑     36.    163.7  บ   
     บ              217.449    137.61         129.           7.67E-01  บ   
     บ    L1O3        4.464    .00635         110. ๑     11.    148.5  บ   
     บ              221.726    138.63         128.           7.78E-01  บ   
     บ    L1M5        3.977    .01022         177. ๑     12.     57.0  บ   
     บ              197.787    132.85         136.           7.03E-01  บ   
     บ                                                                 บ   
     บ  2 Sn-L2                             28030. ๑    208.           บ   
     บ    L2M4        3.663    .84369       23649. ๑    176.    161.8  บ   
     บ              182.353    128.98         141.           6.39E-01  บ   
     บ    L2N4        4.131    .12993        3642. ๑     29.    224.7  บ   
     บ              205.357    134.70         133.           7.30E-01  บ   
     บ    L2M1        3.272    .02078         582. ๑     13.     25.7  บ   
     บ              163.133    124.00         147.           5.40E-01  บ   
     บ    L2N1        4.019    .00560         157. ๑     12.    102.8  บ   
     บ              199.852    133.35         135.           7.11E-01  บ   
     บ                                                                 บ   
     บ  3 Sn-L3                             51092. ๑    234.           บ   
     บ    L3M5        3.443    .83517       42670. ๑    196.      7.7  บ   
     บ              171.539    126.21         144.           5.84E-01  บ   
     บ    L3N5        3.905    .13109        6698. ๑     33.     84.8  บ   
     บ              194.248    131.97         137.           6.90E-01  บ   
     บ    L3M1        3.045    .02549        1303. ๑     14.     46.9  บ   
     บ              151.975    121.02         151.           4.79E-01  บ   
     บ    L3N1        3.793    .00731         373. ๑     12.    221.8  บ   
     บ              188.743    130.60         139.           6.67E-01  บ   
     บ    L3O1        3.923    .00093          48. ๑     12.     56.0  บ   
     บ              195.133    132.19         137.           6.93E-01  บ   
     บ                                                                 บ   
     บ  4 Pile_up                             147. ๑     20.           บ   
     บ    L2O4        7.132    .42457          62. ๑     13.      2.4  บ   
     บ              352.872    166.77         107.           9.39E-01  บ   
     บ    L2O1        6.886    .24165          35. ๑     12.      1.2  บ   
     บ              340.778    164.37         110.           9.33E-01  บ   
     บ    L2M3        7.378    .18649          27. ๑     11.      2.3  บ   
     บ              364.965    169.13         118.           9.45E-01  บ   
     บ    L3M5        7.350    .07841          11. ๑     11.      2.1  บ   
     บ              363.584    168.86         118.           9.44E-01  บ   
     บ    L3M4        7.596    .06888          10. ๑     11.      4.3  บ   
     บ              375.678    171.19         114.           9.49E-01  บ   
     บ                                                                 บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
                                                                           
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ                         FILTER BACKGROUND                       บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
                                                                           
